src/lib/components/mod-grid/mod-grid.svelte

```diff
@@ -167,6 +209,12 @@
 		<div>
 			<CheckboxInput bind:checked={showDetails}>Show details</CheckboxInput>
 		</div>
+		<div>
+			<CheckboxInput bind:checked={hideDLC} onChange={(event) => {
+				if (!event || !event.currentTarget) return;
```

Comment:
pointless check?

Reply:
Yeah I just copied that from elsewhere didnt even know what it did, got rid of it